docs/explanation/ssh.md

```diff
@@ -0,0 +1,111 @@
+(juju-ssh)=
+# Juju SSH
+
+Juju 3.6 introduces a new way to SSH into machines and Kubernetes units.
+
+Instead of connecting directly to machines, SSH connections are now routed through the Juju controller (or via JIMM if used). This centralization improves auditability and security, while keeping the user experience largely unchanged. 
+
+Users can continue to run `juju ssh` or use standard SSH clients.
+
+## Overview
+
+The Juju controller now runs an SSH server capable of terminating SSH connections
+and then forwards the traffic to the target machine or K8s unit.
+
+When a client attempts to SSH to a target unit/machine, the command will resemble the following,
+using OpenSSH syntax.
+
+```
+ssh -J <controller-address>:17022 ubuntu@<unit-number.app.modelUUID>
+```
+
+The ` -J` flag performs an "Jump" connection.
+This is a process that may be familiar to those who connect to machines through a jump/bastion host,
+effectively establishing two SSH connections to reach the target. Supported in some fashion by many SSH clients.
+
+The Juju controller terminates both connections, allowing it to inspect the traffic before
+forwarding it.
+
+## Security
```

Comment:
Wondering about implications for https://documentation.ubuntu.com/juju/3.6/explanation/juju-security/ From what I can see, the changes here don't require any changes there, but please correct me if I'm wrong.